controllers.js

```diff
@@ -1,9 +1,17 @@
-const {selectTopics} = require("./models")
+const {selectTopics, selectArticles} = require("./models")
 
 exports.getTopics = (req,res,next) => {
     selectTopics().then((result)=>{
         res.status(200).send({topics:result})
-        console.log({topics:result})
+        
     })
     .catch(next)
-}
\ No newline at end of file
+}
+
+exports.getArticles = (req,res,next) => {
+    selectArticles().then((result)=>{
+        res.status(200).send({articles:result})
+        console.log({articles:result[0]}, "I'm in the controllers!")
```

Comment:
Please remove console.logs before submitting PRs. The only place that a .log is acceptable (as this is "production" code) is in the 500 internal server error hander.